huaweicloud/services/ga/resource_huaweicloud_ga_accelerator.go

```diff
@@ -0,0 +1,694 @@
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+	"github.com/jmespath/go-jmespath"
+)
+
+func ResourceAccelerator() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceAcceleratorCreate,
+		UpdateContext: resourceAcceleratorUpdate,
+		ReadContext:   resourceAcceleratorRead,
+		DeleteContext: resourceAcceleratorDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(10 * time.Minute),
+			Update: schema.DefaultTimeout(10 * time.Minute),
+			Delete: schema.DefaultTimeout(10 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `Specifies the global accelerator name. The name can contain 1 to 64 characters. Only letters, digits, and hyphens (-) are allowed.`,
+				ValidateFunc: validation.All(
+					validation.StringMatch(regexp.MustCompile(`^[A-Za-z0-9-]+$`),
+						"the input is invalid"),
+					validation.StringLenBetween(1, 64),
+				),
+			},
+			"ip_sets": {
+				Type:     schema.TypeList,
+				MaxItems: 1,
+				Elem:     AcceleratorAccelerateIpSchema(),
+				Required: true,
+				ForceNew: true,
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: `Specifies the description about the global accelerator. The value can contain 0 to 255 characters. The following characters are not allowed: <>`,
+				ValidateFunc: validation.All(
+					validation.StringMatch(regexp.MustCompile(`[^<>]*`),
+						"the input is invalid"),
+					validation.StringLenBetween(0, 255),
+				),
+			},
+			"enterprise_project_id": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Default:     "0",
+				ForceNew:    true,
+				Description: `Specifies the enterprise project ID of the tenant. The value is **0** or a string that contains a maximum of 36 characters in UUID format with hyphens (-). **0** indicates the default enterprise project.`,
+				ValidateFunc: validation.All(
+					validation.StringMatch(regexp.MustCompile(`[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}|0`),
+						"the input is invalid"),
+					validation.StringLenBetween(0, 36),
+				),
+			},
+			"tags": common.TagsForceNewSchema(),
+
+			"status": {
+				Type:     schema.TypeString,
+				Computed: true,
+				Description: `Specifies the provisioning status. The value can be one of the following:
+- **ACTIVE**: The resource is running.
+- **PENDING**: The status is to be determined.
+- **ERROR**: Failed to create the resource.
+- **DELETING**: The resource is being deleted.`,
```

Comment:
Perhaps it would be better to mark the end with two spaces in the .yaml file.
```
Specifies the provisioning status.  
The value can be one of the following:
...
```

Reply:
fixed